### DISCUSSION TITLE
Asking CCF vs Time plot problem

### DISCUSSION THREAD

**Discussion by EvanFadhilahR** [ROBelgium/MSNoise#332] [General]
Dear Thomas,
May i ask problem, i tried to plot CCF vs Time then something like this appears. I check the file directories only like this C:\Msnoise\STACKS\01\001_DAYS\ZZ\PP.TCH02.PA_PP.TCH04.PA. 
![Screenshot 2024-03-12 144518](https://github.com/ROBelgium/MSNoise/assets/163017878/e773635e-a99d-47b7-b739-d8ede5908e24)
how to solve this problem? thankss

**Comment by ThomasLecocq** [comment]
It seems you are using the dev version, for this to work, you need to have adapted the configuration to the latest version (e.g. mov_stack has to be a tuple of tuples ``(('1d','1d'),('2d','1d'))`` would correspond to the previous ``1,2`` - but for this to work you need to rerun the correlations with ``keep_all=Y`` , default in the new version

**Reply by EvanFadhilahR** [reply]
thanks sir
